### ISSUE TITLE
update chart to 10.7.2

### ISSUE THREAD

**Issue by lwolf** [lwolf/gitlab-chart#15]
add support for new params:

[10.5]

* Add GITLAB_UPLOADS_STORAGE_PATH
* Add GITLAB_UPLOADS_BASE_DIR
* Add LDAP_LOWERCASE_USERNAMES

[10.7]

* GITLAB_SIDEKIQ_LOG_FORMAT
* GITLAB_ARTIFACTS_OBJECT_STORE_ENABLED
* GITLAB_ARTIFACTS_OBJECT_STORE_REMOTE_DIRECTORY
* GITLAB_ARTIFACTS_OBJECT_STORE_BACKGROUND_UPLOAD
* GITLAB_ARTIFACTS_OBJECT_STORE_PROXY_DOWNLOAD
* GITLAB_ARTIFACTS_OBJECT_STORE_CONNECTION_PROVIDER
* GITLAB_ARTIFACTS_OBJECT_STORE_CONNECTION_AWS_ACCESS_KEY_ID
* GITLAB_ARTIFACTS_OBJECT_STORE_CONNECTION_AWS_SECRET_ACCESS_KEY
* GITLAB_ARTIFACTS_OBJECT_STORE_CONNECTION_AWS_REGION
* GITLAB_ARTIFACTS_OBJECT_STORE_CONNECTION_AWS_HOST
* GITLAB_ARTIFACTS_OBJECT_STORE_CONNECTION_AWS_ENDPOINT
* GITLAB_ARTIFACTS_OBJECT_STORE_CONNECTION_AWS_PATH_STYLE
* GITLAB_LFS_OBJECT_STORE_ENABLED
* GITLAB_LFS_OBJECT_STORE_REMOTE_DIRECTORY
* GITLAB_LFS_OBJECT_STORE_DIRECT_UPLOAD
* GITLAB_LFS_OBJECT_STORE_BACKGROUND_UPLOAD
* GITLAB_LFS_OBJECT_STORE_PROXY_DOWNLOAD
* GITLAB_LFS_OBJECT_STORE_CONNECTION_PROVIDER
* GITLAB_LFS_OBJECT_STORE_CONNECTION_AWS_ACCESS_KEY_ID
* GITLAB_LFS_OBJECT_STORE_CONNECTION_AWS_SECRET_ACCESS_KEY
* GITLAB_LFS_OBJECT_STORE_CONNECTION_AWS_REGION
* GITLAB_LFS_OBJECT_STORE_CONNECTION_AWS_HOST
* GITLAB_LFS_OBJECT_STORE_CONNECTION_AWS_ENDPOINT
* GITLAB_LFS_OBJECT_STORE_CONNECTION_AWS_PATH_STYLE
* GITLAB_UPLOADS_OBJECT_STORE_ENABLED
* GITLAB_UPLOADS_OBJECT_STORE_REMOTE_DIRECTORY
* GITLAB_UPLOADS_OBJECT_STORE_DIRECT_UPLOAD
* GITLAB_UPLOADS_OBJECT_STORE_BACKGROUND_UPLOAD
* GITLAB_UPLOADS_OBJECT_STORE_PROXY_DOWNLOAD
* GITLAB_UPLOADS_OBJECT_STORE_CONNECTION_PROVIDER
* GITLAB_UPLOADS_OBJECT_STORE_CONNECTION_AWS_ACCESS_KEY_ID
* GITLAB_UPLOADS_OBJECT_STORE_CONNECTION_AWS_SECRET_ACCESS_KEY
* GITLAB_UPLOADS_OBJECT_STORE_CONNECTION_AWS_REGION
* GITLAB_UPLOADS_OBJECT_STORE_CONNECTION_AWS_HOST
* GITLAB_UPLOADS_OBJECT_STORE_CONNECTION_AWS_ENDPOINT
* GITLAB_UPLOADS_OBJECT_STORE_CONNECTION_AWS_PATH_STYLE

**Comment by weisjohn** [comment]
For the record, while I haven't implemented any of these options, in case anyone is wondering, I was able to use that version of `sameersbn/gitlab` by specifying `imageTag: "10.7.2"` in my `config.yml` for `helm install`. Install went through fine.

**Comment by lwolf** [comment]
Yes, it should work with the latest versions, but you can't configure listed flags.
